vscode-eslint/compile-eslint-server.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+# @see https://github.com/microsoft/vscode-eslint
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPO_DIR="${SCRIPT_DIR}"
+SRC_DIR="${REPO_DIR}/checkout"
+SRC_SERVER_DIR="${SRC_DIR}/server"
+DIST_DIR="${REPO_DIR}/out"
+
+
+# ------------------------- #
+# download the source codes #
+# ------------------------- #
+
+pushd "${REPO_DIR}" || exit
+
+rm -rf \
+    "${SRC_DIR}" "${DIST_DIR}" \
+    "package.json" "package-lock.json"
+
+echo 'Enter commit SHA or tag (for example v2.1.0) to build'
+read -p 'SHA or tag: ' ref
```

Comment:
![image](https://user-images.githubusercontent.com/6594915/77012645-51bdbc80-69a9-11ea-86f0-d28df0fe3af2.png)

Btw, but not important since there won't be slashes in `ref`.

```suggestion
read -rp 'SHA or tag: ' ref
```

Reply:
Done